how-to/integrate-with-salesforce/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "openfin-workspace--integrate-with-salesforce",
-  "version": "6.0.0",
+  "version": "5.5.0",
```

Comment:
this should be 6.0.0 to reflect the workspace version.

Reply:
This version number is also used when deploying to github pages.io

Reply:
Missed that, sorry. Updated.